pkg/ssh/ssh.go

```diff
@@ -50,22 +51,30 @@ func SetAuthorizedKeys(cfg *config.CloudConfig, withNet bool) error {
 }
 
 func getKey(key string, withNet bool) (string, error) {
-	if !strings.HasPrefix(key, "github:") {
+	providers := map[string]string{
+		"github": "https://github.com/%s.keys",
+		"gitlab": "https://gitlab.com/%s.keys",
+	}
+
+	url, err := url.Parse(key)
+	if err != nil {
+		return "", err
```

Comment:
We don't want to return this error. Assume an error in parsing means it is not a URL and hence just pass back the `key`.

Reply:
All strings I tested to parse in the playground never returned an error, even invalid urls, so I assumed that parse errors had to be some kind of fatal errors and that's why I chose to return the error here.

I'll change this though.